src/lib/db/event-store.js

```diff
@@ -21,36 +21,35 @@ class EventStore extends EventEmitter {
 
     async store(event) {
         try {
-            await this.db('events').insert({
-                type: event.type,
+            const rows = await this.db('events')
```

Comment:
Not really related to this PR. But we could consider extracting DB names into constants in order to have more robust string handling. Most likely if someone mis-spelled "evets" it would be caught by someone in a review, but why take the chance?

Reply:
True, and it's what we do in most of the other stores, I'll fix it here as well